sdk/client/src/listeners/network-listener/utils/models.ts

```diff
@@ -24,3 +24,21 @@ export interface RequestResponsePair {
 	/** Whether this URL matched a `urlToBlock` so the contents should not be recorded. */
 	urlBlocked: boolean
 }
+
+export interface WebSocketRequest {
+	socketId: string
+	initiatorType: 'WebSocket'
+	type: 'open' | 'close'
+	name: string
+	startTime?: number
```

Comment:
Called this startTime to avoid needing to patch the frontend to handle when sorting, but technically there is no endTime (i.e. this is a timeStamp)

Reply:
Yes, but this would be represented as `endTime` the `close` event that would be sent to the network resources